src/SqlPersistence.Tests/SqlPersistence.Tests.csproj

```diff
@@ -15,6 +15,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="Newtonsoft.Json" Version="[11.0.1, 12.0.0)" />
```

Comment:
Why are these additions necessary? Plus tests would not use the range but 11.0.1 directly

Reply:
Fixed to 11.0.1